packages/cardhost/app/services/css-mode-toggle.js

```diff
@@ -1,10 +1,22 @@
 import Service from '@ember/service';
 import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
 
 export default class CssModeToggleService extends Service {
   @tracked editingCss;
+  @tracked dockLocation = "right";
 
   setEditingCss(value) {
     this.editingCss = value;
   }
+
+  @action
+  dockRight() {
+    this.dockLocation = "right";
+  }
+
+  @action
+  dockBottom() {
+    this.dockLocation = "bottom";
+  }
```

Comment:
This service is now doing more than just handling state of whether or not it's in css editing mode. We should rename this service to something more meaningful. Would love suggestions!

Reply:
I think maybe css-mode-options? But this can be in another PR.